c/datatablemodule.cc

```diff
@@ -193,6 +263,9 @@ void DatatableModule::init_methods() {
   #ifdef DTTEST
     init_tests();
   #endif
+  #ifdef DTDEBUG
```

Comment:
Am I correct, that when one does `make build`, `DTDEBUG` becomes undefined? If so, then what will happen to `make test` in a release mode, if some of the Python tests invoke `get_tracked_objects()`?

Reply:
`get_tracked_objects()` will not be available without the debug mode, you'll get an `AttributeError` exception. So any test that tries to use that method should properly check for its availability. For example, `dt.internal.in_debug_mode()` can help with that.